### PR TITLE
Show user comments

### PR DESCRIPTION
**Pull request by Victoria129**
In this pull request, I added display comments by item.

### CODE REVIEW

File: src/modules/display-comments.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+};
+
+// eslint-disable-next-line
+const createCommentHtml = ({ creation_date, comment, username }) => {
+
+  // eslint-disable-next-line
+  const creationDate = creation_date;
+  const commentli = document.createElement('li');
+  commentli.className = 'comment-li';
+  commentli.innerHTML = `
```
Comment (ernestkorir) [inline src/modules/display-comments.js:20]: Kindly implement the user to be able to input the comments and then to be displayed under comments as the Project requirement state; unable to see the inputs of the message
image

**Review by ernestkorir — CHANGES_REQUESTED** [review]
# Changes Required ♻️
Hi @Victoria129 
Nice job, you are almost there.
image

## (Highlights) Good Points: :
Good implementation of some of the project requirements ✔
GitHub flow was correctly used ✔.
Great design 🔥🔥

**Cheers and Happy coding!**👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

File: src/modules/display-comments.js (hunk 1 above)
Comment (Victoria129) [reply src/modules/display-comments.js:20]: Hi @ernestkorir ,
The ability to add comments is a feature that needs to be implemented after this.
I am going to work on it on the next branch.

**Review by ernestkorir — APPROVED** [review]

**Comment by tugsem** [comment]
## Invalid Code Review Request ❌ 

Hi @Victoria129 @ernestkorir , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<h1>You have submitted a project, that is not for the final review. Please submit the right link for the capstone as requested.</h1>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**